docs/server/source/http-client-server-api.rst

```diff
@@ -265,16 +267,10 @@ Statuses
 
    **Example response**:
 
-   .. literalinclude:: http-samples/get-statuses-block-invalid-response.http
```

Comment:
Before if the block was invalid it only returned the `status`. If it was valid it would return the `status` and `_links`.
Now it always returns only the `status` key so I though the two example responses were redundant.